core/gui/src/app/workspace/service/workflow-result-export/workflow-result-export.service.ts

```diff
@@ -86,9 +86,9 @@ export class WorkflowResultExportService {
    * Export the operator results as files.
    * If multiple operatorIds are provided, results are zipped into a single file.
    */
-  exportOperatorsResultAsFile(download_all: boolean = false): void {
+  downloadOperatorsResultAsFile(downloadAll: boolean = true): void {
```

Comment:
We should use either "export" or "download" and make the name consistent. Your code confuses a lot because of the mixture of "export" and "download". Can you unify the name?

Reply:
I used keyword `download` to separate functions that are related to downloading the result to local from `export` keyword which means transferring them into the database. I believe this is an easier wording, otherwise when someone wants to check downloading option, they get lost in export functionalities and vice versa.

Reply:
Yes, but the UI still shows the word "Export" even though user downloads the result to their local device. The UI and the code is inconsistent with terminologies. If needed, let's ask others' opinion.

Reply:
Its not a big deal but I like to learn perspective of lab mates. @Yicong-Huang if you have time, I appreciate it to chime in and check this. In short, in the UI design, we always use keyword `export` to prevent confusing user (as we discussed last meeting), however, in the code, we have two routes, one is downloading and its functionalities and another one is exporting into the dataset, instead of using keyword `export`, I named core functions for each route separately so we have `downloadOperatorsResultAsFile` and `exportWorkflowExecutionResult` in `workflow-result-export.service.ts`, my intention is to guide future developers easily to find the part they need to work on. Do you think this approach is correct although its inconsistent with the design.

Reply:
Hi @kunwp1, thanks for your patience and catching my naming issue, I learned the rationale behind it and fixed it, PR is ready to merge